desktop/src/main/java/bisq/desktop/primary/overlay/createOffer/amount/AmountController.java

```diff
@@ -118,8 +117,11 @@ public void onActivate() {
 
         baseAmount.setAmount(null);
         if (model.getQuoteSideAmount().get() == null) {
-            //todo we need to adjust to diff fiat exchange rates
-            quoteAmount.setAmount(Fiat.parse("100", model.getMarket().getQuoteCurrencyCode()));
+            // We use 0.004 BTC as default value converted with the price to the fiat amount
+            Quote fixPrice = price.fixPriceProperty().get();
+            Monetary baseCurrencyAmount = Coin.asBtc(400000);
+            Monetary exactAmount = fixPrice.toQuoteMonetary(baseCurrencyAmount);
```

Comment:
This causes NPE when `fixPrice` is null. Happens when amount is not set/deleted by the user.

Reply:
How can you reproduce that?
Here is a fix - assuming its a rare edge case: https://github.com/bisq-network/bisq2/pull/805/commits/b2d5de3dbe4fb823d9d6f34bdc2b6c3f0b7709b1